sharktank/requirements.txt

```diff
@@ -2,8 +2,7 @@ iree-turbine
 
 # Runtime deps.
 gguf==0.6.0
-numpy==1.26.3; sys_platform == 'win32'
-numpy; sys_platform != 'win32'
+numpy==1.26.3
```

Comment:
Must it be specifically `1.26.3`or does `<2` work as well? Or is `1.26.4` already incompatible?

Reply:
<2 seems to work as well:

In fresh venv:
```bash
pip install -r pytorch-cpu-requirements.txt
pip install -r requirements.txt -e sharktank
pip uninstall numpy
pip install "numpy<2"
```

```bash
python -m sharktank.examples.export_paged_llm_v1 --gguf-file=./sglang_integration_tests/llama3.1-8b/llama8b_f16.gguf --output-mlir=./temp/model.mlir --output-config=./temp/config.json --bs=1,4
```

Output:
```
Exporting decode_bs1
Exporting prefill_bs4
Exporting decode_bs4
GENERATED!
Exporting
Saving to './temp/model.mlir'
```

Reply:
Do we wanna leave the version pinned for windows, or set both to `<2`?

Reply:
In that case, let's please not over-strictly pin it to only one specific version and go with `<2.0` or `<2` instead if that#s really needed. However, I assume it's okay to not have it strictly pinned in the CI as well (changed as part of commit 4fcf77b)

Reply:
> Do we wanna leave the version pinned for windows, or set both to `<2`?

Well, I would say let's specify the version range for both. We only need to pin it for `sharktank` not for any use in `shortfin` corret?

Reply:
Set `numpy<2.0` for windows and non-windows